Tests/TestingApps/PaywallsTester/PaywallsTester/UI/Views/UpsellView.swift

```diff
@@ -1,57 +0,0 @@
-//
-//  UpsellView.swift
-//  PaywallsTester
-//
-//  Created by Andrés Boedo on 9/28/23.
```

Comment:
RIP my terrible view, glad to see it vanish

Reply:
Maybe it'll make a comeback as sample code somewhere one day!